coremark/README.md

```diff
@@ -0,0 +1,82 @@
+[![Component Registry](https://components.espressif.com/components/espressif/coremark/badge.svg)](https://components.espressif.com/components/espressif/coremark)
+
+# Coremark for ESP-IDF
+
+This component is a port of [CoreMark® benchmark](https://github.com/eembc/coremark) to ESP-IDF. It handles compiling CoreMark source files, providing necessary functions to measure timestamps, and enables various compiler to get higher performance.
+
+# Using the benchmark
+
+If you want to run the benchmark and see the results, create a demo project from the provided example:
+
+```bash
+idf.py create-project-from-example "espressif/coremark:coremark_example"
+```
+
+You can then build the project in `coremark_example` directory as usual. For example, to build the project for ESP32-S3:
```

Comment:
Maybe we could change this to C3, since the example output is provided for C3 target?